package.json

```
@@ -4,7 +4,8 @@
   "description": "Lightweight and convenient AES encryption/decryption library to make it easier to migrate away from the deprecated crypto-js library",
   "main": "index.js",
   "scripts": {
-    "test": "node --test"
+    "test": "node --test",
+    "test-types": "tsc"
```

With this change, running `npm test` should run both the type check as well as the unit tests locally as well as on CI where just npm test is run
```suggestion
    "test": "npm run test:types && npm run test:unit",
    "test:unit": "node --test",
    "test:types": "tsd"
```